setup.cfg

```diff
@@ -0,0 +1,11 @@
+[flake8]
```

Comment:
![image](https://github.com/user-attachments/assets/275a996d-77bc-4af5-952d-7bb46ec66548)
Requirements.txt
I don't see the usage of ﻿colorama==0.4.6, iniconfig==2.0.0 mccabe==0.7.0 packaging==24.1 pluggy==1.5.0 pycodestyle==2.12.1 pyflakes==3.2.0, please remove redundant packages from the requirements